src/components/MovieCard.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import moment from 'moment'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '../App.css';
+import { Card, ListGroupItem, ListGroup } from "react-bootstrap";
+
+export default function MovieCard(props) {
+
+    //find values using map and prepare for output
+    let htmlPopularMovies = props.fetchedMovies.map((movie) => {
+
+        const renderGenres = genres => {
+            if (!genres.some(genres => genres === undefined)) {
```

Comment:
this is a little kludgey, i think just filter out all the 'undefined' ones first should work fine